invenio_rdm_records/fixtures/tasks.py

```diff
@@ -45,7 +46,10 @@ def get_authenticated_identity(user_id):
 def create_vocabulary_record(service_str, data):
     """Create a vocabulary record."""
     service = current_service_registry.get(service_str)
-    service.create(system_identity, data)
+    try:
+        service.create(system_identity, data)
+    except IntegrityError:
+        current_app.logger.info(f"skipping creation of {data}, already existing")
```

Comment:
Minor: If I understand correctly the new functionality supports 2 scenarios:
1. Add new entries to already existing vocabularies (i.e. those whose vocabulary type is already in db)
2. Recreate (i.e. "reload"), which would delete the entry and allow recreation. 

For case number 2 I wonder if it would be fixed by instead of skipping, making a `service.update` call?

Reply:
Thanks! I've reorganized the code so it now uses `service.update`